ejb3/src/main/resources/org/jboss/as/ejb3/subsystem/LocalDescriptions.properties

```diff
@@ -405,6 +405,7 @@ application-security-domain.security-domain=The Elytron security domain to be us
 application-security-domain.referencing-deployments=The deployments currently referencing this mapping
 application-security-domain.enable-jacc=Enable authorization using JACC
 application-security-domain.http-authentication-factory=The HTTP authentication factory to use for HTTP based invocations
+application-security-domain.legacy-compliant-principal-propagation= Option to switch between legacy compliant principal propagation or Elytron principal propagation
```

Comment:
there is a leading whitespace, which most likely won't matter, but just to follow the existing formatting

Reply:
Fixed